spec/datetimeformat.html

```diff
@@ -103,7 +108,10 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. Let _opt_ be a new Record.
         1. For each row of <emu-xref href="#table-datetimeformat-components"></emu-xref>, except the header row, in table order, do
           1. Let _prop_ be the name given in the Property column of the row.
-          1. Let _value_ be ? GetOption(_options_, _prop_, `"string"`, &laquo; the strings given in the Values column of the row &raquo;, *undefined*).
+          1. If _prop_ is `"fractionalSecondDigits"`, then
+            1. Let _value_ be ? GetNumberOption(options, "fractionalSecondDigits", 0, 3, 0).
```

Comment:
@anba Do you think I should change this to "1. Let _value_ be ? GetNumberOption(options, "fractionalSecondDigits", 1, 3, 0)." instead?

Reply:
I think we want to have `GetNumberOption(options, "fractionalSecondDigits", 1, 3, undefined)`, so that an absent "fractionalSecondDigits" results in using `undefined`. That should result in fewer changes elsewhere, because we no longer need to special-case `0`. (For example in BasicFormatMatcher, see the comment below.)